.github/workflows/build.yml

```diff
@@ -32,16 +32,20 @@ jobs:
     needs: setup
     runs-on: ubuntu-latest
     steps:
-    
+
       # Setup Java 11 environment for the next steps
       - name: Setup Java
         uses: actions/setup-java@v1
         with:
           java-version: 11
-    
+
       - uses: actions/checkout@v2
+
+      - name: Download mock projects
```

Comment:
Should we do this step only during PR and push into the main branch?

Reply:
Mock projects are needed for the tests. I don't see a problem since it takes less than 8s to download the `plugin-utilities` repository with all submodules and tests execute as fast as unit tests.